private/protocol/jsonrpc/unmarshal_test.go

```diff
@@ -1166,6 +1166,26 @@ func TestOutputService3ProtocolTestTimestampMembersCase1(t *testing.T) {
 
 }
 
+func TestOutputService3ProtocolTestTimestampMembersCase2(t *testing.T) {
```

Comment:
These tests are actually generated based on a file in the `model/protocol_tests/output/json.json` file.  To test the `timeFromUnixString` function it would be great to have a unit test in `private/protocol/json/jsonutil/unmarshal_test.go`.  We can later add a protocol test in for fractional time support to the protocol model.